seed/main/cic.xml

```diff
@@ -22,9 +22,9 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		</measurementSystemNames>
 	</localeDisplayNames>
 	<characters>
-		<exemplarCharacters>[a {a\u0331} b c {ch} e f h i {i\u0331} k l {lh} m n {ng} o {o\u0331} p s t w y ʼ]</exemplarCharacters>
-		<exemplarCharacters type="auxiliary">[á d g í j ó q r u v x z]</exemplarCharacters>
-		<exemplarCharacters type="index">[A B C {CH} E F H I K L {LH} M N {NG} O P S T W Y]</exemplarCharacters>
+		<exemplarCharacters>[a {a\u0331} b {ch} f h i {i\u0331} k l {lh} m n {ng} o {o\u0331} p s {sh} t w y ʼ]</exemplarCharacters>
+		<exemplarCharacters type="auxiliary">[á {á\u0331} í {í\u0331} ó {ó\u0331}]</exemplarCharacters>
```

Comment:
Latin script language should have in aux the letters from a-z that aren't in main, eg c.